src/App.tsx

```diff
@@ -3,7 +3,7 @@ import "./App.scss";
 import "bootstrap/dist/css/bootstrap.min.css";
 import Navbar from "./components/Navbar";
 import VideoBG from "./components/background/VideoBG";
-import MainUserPage from "./components/MainUserPage";
+import UserPage from "./components/userPage/UserPage";
```

Comment:
```suggestion
import UserPage from "./components/UserPage/UserPage";
```

be consistent in terms of directories and files namings